src/getting-started/project-setup/using-wasm-pack.md

```diff
@@ -6,12 +6,60 @@ This tool was created by the Rust / Wasm Working Group and is the most actively
 Note that the crate-type in your `Cargo.toml` will need to be `cdylib` when using `wasm-pack`
 {% endhint %}
 
-### Install
+### Install wasm-pack
 
 ```bash
 cargo install wasm-pack
 ```
 
+### Set up your project
+
+{% hint style="info" %}
+If you start from the [yew-wasm-pack-minimal](https://github.com/yewstack/yew-wasm-pack-minimal) template, these setup steps are already taken care of.
```

Comment:
It might be clearer to say "use" rather than "start from"

Reply:
Thank you, updated.